examples/intel/asicworld_lfsr/presynth

```diff
@@ -0,0 +1,121 @@
+#! /usr/local/bin/vvp
+:ivl_version "11.0 (devel)" "(s20150603-462-gfbd69e1-dirty)";
```

Comment:
This obviously is an output file generated by iverilog. We don't include pre-compiled files in the Yosys source tree.